cgmes/cgmes-extensions/src/main/resources/xsd/cgmesMetadataModels.xsd

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xs:schema version="1.0"
+           xmlns:xs="http://www.w3.org/2001/XMLSchema"
+           xmlns:cmm="http://www.powsybl.org/schema/iidm/ext/cgmes_metadata_models/1_0"
+           targetNamespace="http://www.powsybl.org/schema/iidm/ext/cgmes_metadata_models/1_0"
+           elementFormDefault="qualified">
+    <xs:simpleType name='nonEmptyString'>
+        <xs:restriction base='xs:string'>
+            <xs:minLength value='1'/>
+        </xs:restriction>
+    </xs:simpleType>
+    <xs:complexType name="Model">
+        <xs:sequence>
+            <xs:element name="profile" type="cmm:nonEmptyString" maxOccurs="unbounded"/>
+            <xs:element name="dependentOnModel" type="cmm:nonEmptyString" minOccurs="0" maxOccurs="unbounded"/>
+            <xs:element name="supersedesModel" type="cmm:nonEmptyString" minOccurs="0" maxOccurs="unbounded"/>
+        </xs:sequence>
+        <xs:attribute name="part" type="xs:string"/>
+        <xs:attribute name="modelingAuthoritySet" use="required" type="cmm:nonEmptyString"/>
+        <xs:attribute name="id" type="xs:string"/>
+        <xs:attribute name="version" use="required" type="cmm:nonEmptyString"/>
+        <xs:attribute name="description" type="xs:string"/>
+    </xs:complexType>
+    <xs:element name="cgmesMetadataModels">
+        <xs:complexType>
+            <xs:sequence>
+                <xs:element name="model" type="cmm:Model" maxOccurs="unbounded"/>
```

Comment:
Instead of a unbounded list of elements with a `part` enum, it would be clearer to restrict to 4 optional named elements (SV / SSH / TP / EQ).

Reply:
We want the extension to be generic enough to store metadata for other parts. The complete list is defined in `CgmesSubset`. What do you think about defining the part as a simple enum type with the proper values? (like the Side in the branch observability IIDM extension)

```
EQUIPMENT
TOPOLOGY
STATE_VARIABLES
STEADY_STATE_HYPOTHESIS
DYNAMIC
DIAGRAM_LAYOUT
GEOGRAPHICAL_LOCATION
EQUIPMENT_BOUNDARY
TOPOLOGY_BOUNDARY
UNKNOWN
```

Reply:
We are storing in the extension only the information about the models that have been read during the import of the network. We could limit it even more, and store only some of the models we have read.
I think that having a limited list of named elements instead of a property would make future modifications more difficult. What do you think?

Reply:
Ok, I wasn't aware there are more! Ok to keep the list with a `part` enum then.

Reply:
And about defining the `part` as a simple enum type in the xsd, it's a good idea yes

Reply:
the part is now an enum in the xsd